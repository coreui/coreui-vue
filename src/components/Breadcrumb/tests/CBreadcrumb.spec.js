import { mount } from '@vue/test-utils'
import Component from "../CBreadcrumb";

const ComponentName = 'CBreadcrumb'
const wrapper = mount(Component, {
  context: {
    props: {
      items: [{
        text: 'Admin',
        href: '#'
      }, {
        text: 'Manage',
        href: '#'
      }, {
        text: 'Library'
      }]
    }
  }
})

describe(ComponentName, () => {
  it('has a name', () => {
    expect(Component.name).toMatch(ComponentName)
  })
  it('renders correctly', () => {
    expect(wrapper.element).toMatchSnapshot()
  })
});
