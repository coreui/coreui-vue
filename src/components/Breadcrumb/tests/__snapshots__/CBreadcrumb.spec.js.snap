// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CBreadcrumb renders correctly 1`] = `
<ol
  class="breadcrumb"
>
  <li
    class="breadcrumb-item"
    role="presentation"
  >
    <a
      class=""
      href="#"
      target="_self"
    >
      Admin
    </a>
  </li>
  <li
    class="breadcrumb-item"
    role="presentation"
  >
    <a
      class=""
      href="#"
      target="_self"
    >
      Manage
    </a>
  </li>
  <li
    class="breadcrumb-item active"
    role="presentation"
  >
    <span
      class="active"
    >
      Library
    </span>
  </li>
</ol>
`;
